### PR TITLE
made corrections on the HumanEvalComm dataset

### PR DESCRIPTION
**Pull request by DavidAbrahamyan**
We identified some problem descriptions in `human_eval_comm` can be corrected. So we aim to create a v2 version of human_eval_comm by fixing them. This is the 1st PR of the corrections.

### CODE REVIEW

File: Benchmark/HumanEvalComm.json [hunk 1]
```diff
@@ -2,9 +2,10 @@
...
     "prompt1a": "from typing import List\n\n\ndef candidate(numbers: List[float], threshold: float) -> bool:\n    \"\"\" Check if in given list of numbers, are any two numbers meeting a certain condition (e.g. closer to or larger than each other than\n    given threshold).\n    >>> candidate([1.0, 2.0, 3.0], 0.5)\n    False\n    >>> candidate([1.0, 2.8, 3.0, 4.0, 5.0, 2.0], 0.3)\n    True\n    \"\"\"\n",
-    "prompt1c": "from typing import List\n\n\ndef has_close_elements(numbers: List[float], threshold: float) -> bool:\n    \"\"\" Check if in given list of numbers, are any two numbers larger than each other than\n    given threshold.\n    >>> has_close_elements([1.0, 2.0, 3.0], 0.5)\n    False\n    >>> has_close_elements([1.0, 2.8, 3.0, 4.0, 5.0, 2.0], 0.3)\n    True\n    \"\"\"\n",
+    "prompt1c": "from typing import List\n\n\ndef has_close_elements(numbers: List[float], threshold: float) -> bool:\n    \"\"\" Check if in given list of numbers, are any two numbers closer to each other than\n    given threshold.\n    >>> has_close_elements([1.0, 2.0, 3.0], 0.5)\n    True\n    >>> has_close_elements([1.0, 2.8, 3.0, 4.0, 5.0, 2.0], 0.3)\n    False\n    \"\"\"\n",
```
Comment (jie-jw-wu) [inline Benchmark/HumanEvalComm.json:5]: revert this change to make 1c diverse

File: Benchmark/HumanEvalComm.json
Comment (jie-jw-wu) [inline Benchmark/HumanEvalComm.json:1]: Could you change the file name to something like `Benchmark/HumanEvalComm_v2.json`? (please ensure the PR displays your change diff instead of displaying a whole new file for HumanEvalComm_v2.json)

Besides, please add a copy of this file before (without) your changes as `Benchmark/HumanEvalComm_v1.json`, as some of our previous experiments were using v1 json.

Otherwise the PR looks good to me.

**Review by jie-jw-wu — APPROVED** [review]
approved with 2 comments. thanks!

**Comment by jie-jw-wu** [comment]
@DavidAbrahamyan could you merge this PR? (click "Merge pull request" button). After that, this PR change will go into the master branch. Then, you could pull the lastest master to get this change in your 2nd PR.

**Comment by DavidAbrahamyan** [comment]
Merged!
